feature/login/src/main/java/com/iteneum/login/presentation/LoginView.kt

```diff
@@ -0,0 +1,182 @@
+package com.iteneum.login.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import com.iteneum.designsystem.components.LpFilledTonalButton
+import com.iteneum.designsystem.components.LpOutlinedTextFieldMail
+import com.iteneum.designsystem.components.LpOutlinedTextFieldPassword
+import com.iteneum.designsystem.R.drawable.*
+import com.iteneum.login.R
+
+
+@Composable
+fun LoginView() {
+    ConstraintLayout(
+        modifier = Modifier
+            .fillMaxSize()
+
+    ) {
+        val (logo, mail, password, tonalButton, loginWith, socialNetworkLogins, registerButton) = createRefs()
+
+        Image(
+            painter = painterResource(id = leasepertlogo),
+            contentDescription = stringResource(R.string.lv_logo),
+            modifier = Modifier
+                .size(143.dp, 142.dp)
+                .constrainAs(logo) {
+                    top.linkTo(parent.top, margin = 50.dp)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                }
+        )
+        LpOutlinedTextFieldMail(
+            modifier = Modifier
+                .constrainAs(mail) {
+                    top.linkTo(logo.bottom, margin = 30.dp)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                },
+            label = stringResource(R.string.lv_email),
+            isValid = false,
+            supportTextError = stringResource(R.string.lv_support_text_error),
+            onValueChange = {}
+        )
+        LpOutlinedTextFieldPassword(
+            modifier = Modifier
+                .constrainAs(password) {
+                    top.linkTo(mail.bottom, margin = 2.dp)
+                    start.linkTo(mail.start)
+                    end.linkTo(mail.end)
+                },
+            onPasswordChange = {},
```

Comment:
Please add a comment TODO of what is going to be the functionality for this `onPasswordChange`